packages/notification/test/notification.test.js

```diff
@@ -57,6 +57,34 @@ describe('vaadin-notification', () => {
       expect(isVisible(document.body.querySelector('vaadin-notification-container'))).to.be.true;
     });
 
+    it('should cancel vaadin-overlay-close events when the source event occurred within the container', (done) => {
+      listenOnce(document, 'click', (clickEvent) => {
```

Comment:
Would this work?

```js
const clickEvent = click(notification._card);
const overlayCloseEvent = fire(document, 'vaadin-overlay-close', { sourceEvent: clickEvent }); 
expect(overlayCloseEvent.defaultPrevented).to.be.true;
```

The `click` and `fire` are from `@vaadin/testing-helpers`.

Reply:
Unfortunately not, the clickEvent does not have a composedPath in that case, which is required for the logic that cancels the event.

Reply:
Got it. 👍